### PR TITLE
Update Index_page.jsp

### PR DESCRIPTION
**Pull request by VinothKumar-P**
Changed in Functional UI

### CODE REVIEW

File: WebContent/jsp/Index_page.jsp [hunk 1]
```diff
@@ -42,13 +42,13 @@
                         <a href="viewIndex.do">
                         
                             <i class="material-icons">dashboard</i>
-                            <p>Dashboard</p>
+                            <p>Dashboard For Asset</p>
```
Comment (VinothKumar-P) [inline WebContent/jsp/Index_page.jsp:45]: Added

File: WebContent/jsp/Index_page.jsp [hunk 2]
```diff
@@ -42,13 +42,13 @@
...
                         </a>
                     </li>
                      <li>
                         <a href="Master_list.do">
                             <i class="material-icons">assignment</i>
-                            <p>Master Asset List</p>
+                            <p>Master Asset List2</p>
```
Comment (VinothKumar-P) [inline WebContent/jsp/Index_page.jsp:51]: Added

**Review by VinothKumar-P — COMMENTED** [review]
Added

**Review by beamboy94 — APPROVED** [review]
Added

**Review by beamboy94 — APPROVED** [review]
Added

**Pull request merged**